### ISSUE TITLE
Turn solver

### ISSUE THREAD

**Issue by 1R0NYMAN** [bupticybee/TexasSolver#141]
Does TexasSolver provide functionality for solving only specifitc turn and river?

**Comment by bupticybee** [comment]
Of course, just select 4 or 5 public card and the solver will automaically switch to turn/river mode.
